scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -463,3 +522,29 @@ task PlotACNVResults {
         #corresponding output images
     }
 }
+
+#Calling ACNV segments as CNLoH and balanced
+task CNLoHAndSplitsCaller {
+    String entity_id
+    File jar_file
```

Comment:
It would make sense to rename this `gatk_jar` or `gatk_protected_jar` or something like that in every task of the workflow.